utils.py

```diff
@@ -0,0 +1,16 @@
+def transform_polygons_to_multipolygon(polygons):
```

Comment:
Might be useful to add a comment that we are talking geojson format, rather than WKT or any other format for multipolygons.